src/tokens/TokenBalances.tsx

```diff
@@ -127,6 +149,15 @@ function TokenBalancesScreen({ navigation }: Props) {
     })
   }
 
+  const handleMeasureBalanceHeight = (event: LayoutChangeEvent) => {
+    balanceHeightRef.current = event.nativeEvent.layout.height
+  }
+
+  const handleScroll = (event: { nativeEvent: NativeScrollEvent }) => {
+    const opacityValue = event.nativeEvent.contentOffset.y > balanceHeightRef.current ? 1 : 0
+    headerOpacity.value = withTiming(opacityValue)
+  }
```

Comment:
We should use [`useAnimatedScrollHandler`](https://docs.swmansion.com/react-native-reanimated/docs/api/hooks/useAnimatedScrollHandler) along with `Animated.FlatList` otherwise the trigger won't be driven by the native driver.
i.e. it will be delayed if the JS thread is busy.

Reply:
thanks! i've updated this :D